.github/workflows/ci.yml

```
@@ -16,116 +19,120 @@ jobs:
+        working-directory: backend
+        run: |
+          npm run build
+
   deploy:
     runs-on: ubuntu-latest
     needs: build
-    if: github.event_name == 'pull_request'
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)